src/components/ha-bar.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+
+import {
+  getValueInPercentage,
+  normalize,
+  roundWithOneDecimal,
+} from "../util/calculate";
+
+@customElement("ha-bar")
+export class HaBar extends LitElement {
+  @property({ type: Number }) public min = 0;
+
+  @property({ type: Number }) public max = 100;
+
+  @property({ type: Number }) public value!: number;
+
+  @property({ type: Number }) public target?: number; // Number between 0 and 100
```

Comment:
Can we remove target and have the color handle by the user of the component? That allows us to have multiple points (green, yellow, red). Let's align it as much as possible with the gauge, as they are the same but in a different format.